build/templates/_converters.py.mako

```diff
@@ -266,6 +266,12 @@ def convert_month_to_timedelta(months):
     return hightime.timedelta(days=(30.4167 * months))
 
 
+# Scaling factor to apply on seconds to get months
+# would be 1/(60 seconds * 60 minutes * 24 hours * 30.4167 days)
+def convert_timedelta_to_months_int32(value):
+    return _convert_timedelta(value, _visatype.ViInt32, 1.0 / (60 * 60 * 24 * 30.4167))
```

Comment:
I don't feel great about us coming up with some arbitrary definition for months.
There's a good reason that datetime.timedelta (and, therefore, hightime) doesn't support it. It's not a standard length of time.

Depending on how this is used, it could lead to confusion or issues filed.

Why do you want to add this?